app/forms/nsm/search_form.rb

```diff
@@ -0,0 +1,16 @@
+module Nsm
+  class SearchForm < ::SearchForm
+    def statuses
+      [show_all] + %i[
+        not_assigned
+        in_progress
+        provider_updated
```

Comment:
NSM also doesn't use "provider_updated" or "expired" statuses, so you can remove those